source/guide_jingo.rst

```diff
@@ -0,0 +1,257 @@
+  Cloning into '/home/isabell/jingo'...
+  remote: Counting objects: 2684, done.
+  remote: Total 2684 (delta 0), reused 0 (delta 0), pack-reused 2684
+  Receiving objects: 100% (2684/2684), 1.88 MiB | 3.19 MiB/s, done.
+  Resolving deltas: 100% (1356/1356), done.
+  [isabell@stardust ~]$
+
+
+Then you need to install the dependencies using the node packet manager:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd ~/jingo
+  [isabell@stardust jingo]$ npm install
+  [...]
+  added 626 packages in 17.17s
+  [isabell@stardust jingo]$
+
+
+Configuration
+=============
+
+Data storage
+------------
+
+Jingo uses a Git repository to manage its data, you just need to create an empty folder and initialize Git within it:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mkdir ~/jingo_data
+  [isabell@stardust ~]$ cd ~/jingo_data
+  [isabell@stardust jingo_data]$ git init
+  Initialized empty Git repository in /home/isabell/jingo_data/.git/
+  [isabell@stardust jingo_data]$
+
+You then have to configure the Git repository with name and email, if you haven't done it already globally:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd ~/jingo_data
+  [isabell@stardust jingo_data]$ git config user.name "$USER"
+  [isabell@stardust jingo_data]$ git config user.email "$USER@uber.space"
+  [isabell@stardust jingo_data]$
+
+.. note:: You can of course set arbitrary informations here. If you want to share your data using Github for example, you might want to set the appropriate data.
+
+
+Configure port
+--------------
+
+Since Node.js applications use their own webserver, you need to find a free port and bind your application to it.
+
+.. include:: includes/generate-port.rst
+
+Change the configuration
+------------------------
+
+You first need to create a default config file using the ``jingo`` command:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ ~/jingo/jingo -s > ~/jingo/config.yaml
+  [isabell@stardust ~]$
+
+Then you can adjust this file ``~/jingo/config.yaml`` with your settings. First of all, set the correct port and the repository folder. Be sure to replace ``<username>`` and ``<yourport>`` with the appropriate data:
+
+.. code-block:: yaml
+  :emphasize-lines: 6,11
+
+  # Configuration sample file for Jingo (YAML)
+  application:
+    title: Jingo
+    logo: ''
+    favicon: ''
+    repository: '/home/<username>/jingo_data'
+    docSubdir: ''
+  [...]
+  server:
+    hostname: localhost
+    port: <yourport>
+    localOnly: false
+    baseUrl: ''
+  [...]
+
+You will also need to set up an account to login. You can choose multiple ways for authentication like *Github*, *Google*, *Ldap*.
+For the initial config here, the most simple way is to set up a local account that is stored in the config file. You first need to choose a password and create a hash:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ ~/jingo/jingo --hash-string MySuperSecretPassword
+  1531379b51fd883c2eee31a7fa95ae9a94e0e1bf
```

Comment:
Haven't tested it but if that's the hash for `MySuperSecretPassword` could you please replace it with random data? If you already did: nevermind ☺️ We want to make sure that the password is *not* `MySuperSecretPassword`

Reply:
ok, I hope now its made really sure ;)